server/controllers/favourites.js

```diff
@@ -0,0 +1,47 @@
+import models from '../models';
+const favourite = models.Favourite;
+
+
+const FavouriteRecipe = {
+  addFavourite(req, res) {
+    return favourite
+    .create({
+      recipeId: req.body.recipeId,
+      userId: req.body.userId
+    })
+    .then(() => {
+      res.send ({
+        status: 'Done',
+        message: 'Favourite Added'
+      });
+
+    })
+    .catch (error => res.status(400).send ({
+      status: false,
+      message: error
+    }));
+  },
+
+
+  getFavourite(req, res){
+    return favourite
+    .findAll({
+    where: {
+      userId: req.params.userId
+    },
+    })
+    .then((favourite) => {
+      if(favourite.length < 1){
+        return res.status(200).send({
```

Comment:
Http response code `200` does not correspond with scenario. Checkout Http response code `404`